app/controllers/coronavirus/reorder_announcements_controller.rb

```diff
@@ -24,10 +24,10 @@ def update
       end
 
       if success
-        message = "Announcements were successfully reordered."
+        message = I18n.t("coronavirus.pages.announcements.reorder.success")
         redirect_to coronavirus_page_path(page.slug), notice: message
       else
-        message = "Sorry! Announcements have not been reordered: #{draft_updater.errors.to_sentence}."
+        message = I18n.t("coronavirus.pages.announcements.delete.failed", error: draft_updater.errors.to_sentence)
```

Comment:
Ah good spot! You caught me in a copy and paste error. I can add a commit later that `!`'s the coronavirus content. Thanks.

Reply:
We're already setting [config.action_view.raise_on_missing_translations = true](https://github.com/alphagov/collections-publisher/blob/master/config/application.rb#L27) 🎉

Reply:
Oh great - you might find it reads nicer in controllers to use the helper from the view with `helpers.t` rather than `I18n.t!` it feel a bit more consistent with the views (but would still need I18n for tests)

Reply:
I've never seen `helpers.t` used anywhere else. Is that a new standard?

Reply:
In Rails 5 [the ability to call helpers in controllers](https://bigbinary.com/blog/rails-add-helpers-method-to-ease-usage-of-helper-modules-in-controllers) was added. So instead of having to do `include MyHelper` you can access them directly from a helper object.

There isn't any specific standard for using `helpers.t` I'm just theorising that this would be an easier read as there is symmetry between controllers with `helpers.t("key") and views with `<%= t("key") %>`. I feel that having `I18n.t!("key") in controllers, while views have `<%= t("key") %>`, makes you wonder why one is a bang and the other isn't.

Reply:
Do we still need the `!` if we're already raising an error on missing translation?

Reply:
Yup. `I18n.t` doesn't make any use of `config.action_view.raise_on_missing_translations` that configuration value only applies when you use Rails' `t` helper.